pkg/plugin/predicates/predicate_manager_test.go

```diff
@@ -69,8 +69,8 @@ func TestPreemptionPredicatesEmpty(t *testing.T) {
 	node := framework.NewNodeInfo()
 	node.SetNode(&v1.Node{})
 	victims := make([]*v1.Pod, 0)
-	index, ok := predicateManager.PreemptionPredicates(pod, node, victims, 0)
-	assert.Check(t, !ok, "check should have failed")
+	index := predicateManager.PreemptionPredicates(pod, node, victims, 0)
+	assert.Check(t, index == -1, "check should have failed")
 	assert.Equal(t, index, -1, "wrong index")
```

Comment:
Only 1 assert required. 

-> assert.Equal(t, index, -1, "should not find a victim after preemption check.")